parallel-install/example/example.go

```diff
@@ -30,8 +29,9 @@ func main() {
 
 	log := logger.NewLogger(*verbose)
 
-	if kubeconfigPath == nil || *kubeconfigPath == "" {
-		log.Fatal("kubeconfig is required")
+	if (kubeconfigPath == nil || *kubeconfigPath == "") &&
+		(kubeconfigContent == nil || *kubeconfigContent == "") {
```

Comment:
I don't think you need to check for `nil` here. The `flag.String()` function will always return a non `nil` string pointer (either passed flag or pointer to empty string). In this case if the flag is not passed, an empty string, but the pointer will never be nil.

Reply:
Actually it is just in `example.go` and the `version` is checked in the same manner. Is it a problem to leave it this way here?

Reply:
Not a problem, just less code => easier to read.